docs/usage/configuration-options.md

```diff
@@ -205,6 +205,16 @@ So for example you could choose to automerge all (passing) `devDependencies` onl
     Renovate tries to delay until Azure is in the expected state, but it will continue if it takes too long.
     In some cases this can result in a dependency not being merged, and a fresh PR being created for the dependency.
 
+### Automerge and GitHub branch protection rules
+
+You must select at least one status check in the **Require status checks to pass before merging** section of your branch protection rules on GitHub, if you match all three conditions:
+
+- `automerge=true`
+- `platformAutomerge=true`, Renovate defaults to `true`
+- you use use GitHub's **Require status checks to pass before merging** branch protection rule
+
+If you don't select any status check, and you use platform automerge, then GitHub might automerge PRs with failing tests!
```

Comment:
I wanted to have a proper section for the advice, so I can use a list. We already have a lot of admonitions in the `automerge` docs. I don't think adding another colored box is going to help readers.

But my new `h3` section is causing a test to fail:

```
 documentation › website-documentation › configuration-options › has headers for every required sub-option

expect(received).toEqual(expected) // deep equality

- Expected  - 0
+ Received  + 1

@@ -1,6 +1,7 @@
  Array [
+   "Automerge and GitHub branch protection rules",
    "abortIgnoreStatusCodes",
    "abortOnError",
    "allowedVersions",
    "artifactAuth",
    "authType",

    at Object.<anonymous> (test/documentation.spec.ts:60:28)
```

Reply:
Unfortunately you can't use headings, as they have special meanings